README.md

```diff
@@ -1,24 +1,73 @@
-# README
-
+## Zaka
 This README would normally document whatever steps are necessary to get the
 application up and running.
-
+> A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 Things you may want to cover:
+## Built With
 
 * Ruby version
+- Ruby
+- Ruby on Rails
+- PostgreSQL
 
 * System dependencies
+## Getting Started
 
 * Configuration
+To get a local copy up and running follow these simple example steps.
 
 * Database creation
+1- Clone `git clone https://github.com/HSMathebula/budget.git` <br>
+2- In your terminal go to the project name by `cd budget`. <br>
+3- Run`rails s`. <br>
+4- Run `rails db:create` to create database.
 
 * Database initialization
+To setup linters on your local environment, please follow these steps.
 
 * How to run the test suite
+1- Run `bundle install`. <br>
+2- Run `rubocop .` for the errors.
 
 * Services (job queues, cache servers, search engines, etc.)
+To run the tests <br>
 
 * Deployment instructions
+Run `rspec` on terminal window
 
 * ...
+### Prerequisites
+
+IDE(VSCode etc.), WSL(for Windows OS), for linters RuboCop
+
+## Author
+
+👤 **Happen Mathebula**
+
+- GitHub: [@HSMathebula](https://github.com/HSMathebula)
+- Twitter: [@happen_sipho](https://twitter.com/HappenSipho)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/happen-sipho-mathebula)
+
+
+## Attributions
+
```

Comment:
- [ ] [OPTIONAL] Great implementation @HSMathebula 👍🏽  however, for proper project documentation, kindly add the link for the project deployment and the project video presentation to the README file. It is also good as you have attached the links to the PR description.